docs/tech-preview-blog-post.md

```diff
@@ -0,0 +1,128 @@
+
+But even in English, there are grammatical variations required:
+
+> Your item had _no views_...
+> 
+> Your item had 1 _view_...
+> 
+> Your item had 1,043 _views_...
+
+Once messages have been created, they need to be translated into the various
+languages and adapted for the various cultures around the world.
+Previously there was not widely adopted standard
+and existing formats provided only rudimentary support for managing
+the variations needed by other languages,
+it could be difficult for translators to do their work effectively.
+
+For example, the same message shown above, needs a different set of variations
+in order to support Polish:
+
+> Twój przedmiot nie _ma_ żadnych _wyświetleń_.
+> 
+> Twój przedmiot _miał_ 1 _wyświetlenie_.
+> 
+> Twój przedmiot _miał_ 2 _wyświetlenia_.
+> 
+> Twój przedmiot _ma_ 5 _wyświetleń_.
+
+
+MessageFormat 2 makes it easy to write messages like this
+without developers needing to know about such language variation.
+In fact, developers don't need to learn about any of the language
+and formatting variations needed by languages other than their own
+nor write code that manipulates formatting.
+
+MessageFormat 2 messages can be simple strings:
+```
+    Hello, world!
+```
+
+A message can also include _placeholders_ that are replaced by user-provided values:
+```
+    Hello {$user}!
+```
+
+The user-provided values can be transformed or formatted using functions:
+```
+    Today is {$date :date}
+    Today is {$date :datetime weekday=long}.
+```
+
+Messages can use a function (called a _selector_) to choose between different
+different versions of a message.
+These allow messages to be tailored to the grammatical (or other) requirements of 
+a given language:
+```
+    .match {$count :integer}
+    0   {{You have no views.}}
+    one {{You have {$count} view.}}
+    *   {{You have {$count} views.}}
+```
+
+Unlike previous versions of MessageFormat, MessageFormat 2 is designed for
+extension by implementers and even end users.
+This means that new functionality can be added to messages without modifying
+either existing messages or, in some cases, even the core library containing the 
+MessageFormat 2 code.
+
+MessageFormat 2 provides a rich and extensible set of functionality
+to permit the creation of natural-sounding, grammatically-correct, 
+messages, while enabling rapid, accurate translation
+and extension using new and improved internationalization functionality
+in any computing system.
+
+The Technical Preview is available for comment.
+The stable version of this specification is expected to be part of the 
+Fall 2024 release of CLDR (v46).
+Implementations are available in ICU4J (Java) and ICU4C (C/C++)
+as well as JavaScript.
+Feedback about implementation experience,
+syntax,
+functionality,
+or other parts of the specification is welcome!
+See the end of this article for details on participation and how to comment on this work.
+
+MessageFormat 2 consists of multiple parts: 
```

Comment:
Seems like this para should be moved before the para beginning "The Technical Preview..."?